src/database/schema.js

```diff
@@ -150,7 +150,7 @@ export const schema = {
     StocktakeBatch,
     User,
   ],
-  schemaVersion: 18,
+  schemaVersion: 6,
```

Comment:
@Chris-Petty - Will has done a PR which has been merged that increments to 6 - I just leave it there rather than to 7? Only increment develop after releasing an update?

Reply:
Yea you're correct, ideally just increment once for the formal release.